services/src/main/scala/cromwell/services/auth/impl/GithubAuthVendingActor.scala

```diff
@@ -25,27 +27,32 @@ class GithubAuthVendingActor(serviceConfig: Config, globalConfig: Config, servic
 
   lazy val enabled: Boolean = serviceConfig.getBoolean("enabled")
 
-  private lazy val ecmConfigOpt: Option[EcmConfig] = EcmConfig.apply(serviceConfig)
+  lazy val ecmConfigOpt: Option[EcmConfig] = EcmConfig.apply(serviceConfig)
   lazy val ecmServiceOpt: Option[EcmService] = ecmConfigOpt.map(ecmConfig => new EcmService(ecmConfig.baseUrl))
 
   override def receive: Receive = {
-    case GithubAuthRequest(userToken) if enabled =>
+    case GithubAuthRequest(terraToken) if enabled =>
       val respondTo = sender()
       ecmServiceOpt match {
         case Some(ecmService) =>
-          ecmService.getGithubAccessToken(userToken) onComplete {
-            case Success(githubToken) => respondTo ! GithubAuthTokenResponse(githubToken)
+          ecmService.getGithubAccessToken(terraToken.value) onComplete {
```

Comment:
Should the `ecmService.getGithubAccessToken` be upgraded to take a TerraToken too? ie only unpack it at the very last moment?

Reply:
done!